CHANGELOG.md

```diff
@@ -1,5 +1,32 @@
 # Changelog
 
+## [0.12.2] - TBD
+
+### Added
+- AWS KMS server Keymanager plugin (#2066)
+- Upstream Authority plugin that uses the Certificate Authority Service from Google Cloud Platform (#2172)
+- Improved error returned during attestation of agents (#2159)
+- AWS IID node attestor now supports running in a location with no public internet access available for the server (#2119)
+- The k8s notifier can now rotate Webhook CA Bundles (#2022)
+- Rate limiting on X509 signing and JWT signing can now be disabled (#2142)
+- Uptime metrics in server and agent (#2032)
+- Timeouts to calls to KeyManager plugins (#2044)
```

Comment:
How long is the timeout? Might be useful to know in advance of the upgrade to predict a potential issue e.g. Calls to KeyManager plugins now time out at 30 seconds